datatype/pluto_error/code.go

```diff
@@ -0,0 +1,16 @@
+package pluto_error
+
+import "net/http"
+
+var (
+	ServerError           = 1000
+	BadRequest            = NewPlutoError(http.StatusBadRequest, 1001, "Bad Request", nil)
+	MailIsAlreadyRegister = NewPlutoError(http.StatusForbidden, 1002, "Mail is already been registered", nil)
+	MailIsNotExsit        = NewPlutoError(http.StatusForbidden, 1003, "Mail is not exist", nil)
+	InvalidPassword       = NewPlutoError(http.StatusForbidden, 1004, "Invalid Password", nil)
+	InvalidRefreshToken   = NewPlutoError(http.StatusForbidden, 1005, "Invalid Refresh Token", nil)
```

Comment:
Can we divide the error code by types?
Just like
```
User: 10xx
Token: 20xx
```

Reply:
I can't figure out a better way to do this since we generally have ServerError and BadRequest.

Reply:
:god-fan: